modules/guides/pages/bulk-operations.adoc

```diff
@@ -0,0 +1,420 @@
+= Bulk Operations
+:description: How to perform bulk CRUD operations in Couchbase.
+:page-pagination: prev 
+:page-topic-type: guide
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+include::partial$example-attributes.adoc[]
+
+[abstract]
+{description}
+
+Performing bulk operations with Couchbase Server can be done in several ways, depending on the SDK or command line tool used to perform them.
+In the following sections you will find some basic examples of how to perform bulk CRUD 
+operations on Couchbase documents.
+
+== Creating Multiple Documents
+
+To create multiple documents in Couchbase perform a bulk `insert` operation.
```

Comment:
Remove markup from `insert`, `get`, `upsert`, `replace`, `remove` throughout this page when used as generic terms rather than specific commands or options.